eng/pipelines/runtime.yml

```diff
@@ -147,7 +148,6 @@ extends:
           - windows_x86
           - windows_arm
           - windows_arm64
-          - FreeBSD_x64
```

Comment:
Is this intentional?

Reply:
I initially modeled what Eduardo did in #102681 to reduce the size of `runtime.yml` by moving some builds to other templates, but after adding the FreeBSD builds back, I'm not getting any complaints about overly large YMLs. I reverted this.